ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -94,15 +94,6 @@ const updateTaskGroupIdHistory = id => {
     '&:hover': {
       background: fade(theme.palette.primary.main, 0.9),
     },
-    '& $input': {
```

Comment:
This is still needed otherwise we would lose all of the styling. Instead, we could probably do:

```diff
diff --git a/ui/src/views/Tasks/TaskGroup/index.jsx b/ui/src/views/Tasks/TaskGroup/index.jsx
index 617a96a5d..7eb3e0418 100644
--- a/ui/src/views/Tasks/TaskGroup/index.jsx
+++ b/ui/src/views/Tasks/TaskGroup/index.jsx
@@ -94,7 +94,7 @@ const updateTaskGroupIdHistory = id => {
     '&:hover': {
       background: fade(theme.palette.primary.main, 0.9),
     },
-    '& $input': {
+    '& input': {
       transition: 'unset !important',
       width: 'unset !important',
       color: fade(theme.palette.text.primary, 0.5),
```

Reply:
Fixed!

Thanks :-)